src/components/Speciality/Training.js

```diff
@@ -5,14 +5,14 @@ import { Padding } from 'styled-components-spacing'
 import GreyBackground from '../Common/GreyBackground'
 import StyledLink from '../Common/StyledLink'
 
-const TrainingStage = ({ title, body, icon }) => (
+const TrainingStage = ({ title, body, iconTitle, iconUrl }) => (
   <Col width={[1, 1, 1, 1, 4 / 12]}>
     <Padding top={4} bottom={4}>
       <Padding bottom={1.5}>
-        <img src={`https://${icon.file.url}`} alt={icon.title} />
+        <img src={`https://${iconUrl}`} alt={iconTitle} />
```

Comment:
I'm so confused as to why we're writing `https://` in here 🤔 this should be removed. For example, what if the url changes to `https://path.to.image` and we then get `https://https://` 👎

Reply:
Again this should also use the image component from Common and the `GatsbyContentfulFluid_withWebp`

Reply:
done.